SU-HSDO/suhumsci
# Feat(STN-858):  private collections styling

# READY FOR REVIEW

## Summary
Locate references to classes specifically targeting Collections (`.ptype-hs-collection`, `.field-hs-page-components`) and/or classes that are represented differently between public/private collections. The 2 shown above were all I was able to locate and were cases where I simply added the private collection class to the list (`.ptype-hs-priv-collection`, `.field-hs-priv-page-components`).

### Notes:
I do not know if  `su_humsci_theme` gets updates, but I added to it in this case for completeness. I am open to dropping this commit if it is unnecessary.

## Issue
[STN-858](https://sparkbox.atlassian.net/browse/STN-858)

## Urgency
_Medium_

## Steps to Test
Testing this can likely be time-consuming, though the amount of components available in a private collection are fairly limited. I will include some screenshots displaying private vs. public collections. The only difference I was able to find was that Private collections did not have the correct spacing that `ptype-hs-collection` was receiving.
1.) Create private page
2.) Create public page
3.) Add Collection(s) to private page and utilize all the available components.
4.) Repeat step #3 for your public page.
5.) Compare the 2
6.) Repeat process for unchecked theme.
7.) Insert any other testing methods you see fit.



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
